src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -0,0 +1,460 @@
+            {t('earnFlow.poolInfoScreen.ageOfPool')}
+          </Text>
+          <Touchable onPress={infoIconPress} borderRadius={24}>
+            <InfoIcon size={16} color={Colors.gray3} />
+          </Touchable>
+        </View>
+        <Text style={styles.cardTitleText}>{formattedAge(dateInterval)}</Text>
+      </View>
+    </Card>
+  )
+}
+
+function LearnMoreTouchable({
+  manageUrl,
+  providerName,
+}: {
+  manageUrl: string
+  providerName: string
+}) {
+  const { t } = useTranslation()
+  return (
+    <View style={styles.learnMoreContainer}>
+      <Touchable borderRadius={8} onPress={() => navigateToURI(manageUrl)}>
+        <View style={styles.learnMoreView}>
+          <OpenLinkIcon color={Colors.black} size={24} />
+          <Text style={styles.learnMoreText}>
+            {t('earnFlow.poolInfoScreen.learnMoreOnProvider', { providerName })}
+          </Text>
+        </View>
+      </Touchable>
+    </View>
+  )
+}
+
+function ActionButtons({ earnPosition }: { earnPosition: EarnPosition }) {
+  const { bottom } = useSafeAreaInsets()
+  const insetsStyle = {
+    paddingBottom: Math.max(bottom, Spacing.Regular16),
+  }
+  const { t } = useTranslation()
+  const { availableShortcutIds } = earnPosition
+  const deposit = availableShortcutIds.includes('withdraw')
+  const withdraw = availableShortcutIds.includes('withdraw')
+
+  return (
+    <View style={[styles.buttonContainer, insetsStyle]}>
+      {withdraw && (
+        <Button
+          text={t('earnFlow.poolInfoScreen.withdraw')}
+          onPress={() => {
+            // TODO (ACT-1343): EarnCollectScreen should take earnPositon instead of depositTokenId and poolTokenId
+            // navigate(Screens.EarnCollectScreen, { earnPosition })
+            Logger.debug('Withdraw Button Pressed!')
+          }}
+          size={BtnSizes.FULL}
+          type={BtnTypes.SECONDARY}
+          style={styles.flex}
+        />
+      )}
+      {deposit && (
+        <Button
+          text={t('earnFlow.poolInfoScreen.deposit')}
+          onPress={() => {
+            // TODO hook up after ACT-1342 is merged
+            // navigate(Screens.EarnEnterAmount, { pool: earnPosition })
+            Logger.debug('Deposit Button Pressed!')
+          }}
+          size={BtnSizes.FULL}
+          style={styles.flex}
+        />
+      )}
+    </View>
+  )
+}
+
+type Props = NativeStackScreenProps<StackParamList, Screens.EarnPoolInfoScreen>
+
+export default function EarnPoolInfoScreen({ route, navigation }: Props) {
+  const { pool } = route.params
+  const { networkId, tokens, displayProps, appName, dataProps } = pool
+  const allTokens = useSelector((state) => tokensByIdSelector(state, [networkId]))
+  const tokensInfo = useMemo(() => {
+    return tokens
+      .map((token) => allTokens[token.tokenId])
+      .filter((token): token is TokenBalance => !!token)
+  }, [tokens, allTokens])
+
+  // console.log('Tom - earnPosition.dataProps.yieldRates: ', pool.dataProps.yieldRates)
```

Comment:
remove

Reply:
Removed in df6bd81fef31ce7d8ba28534f62cd6006c12debc.